build-config/make/u-boot.make

```diff
@@ -46,6 +46,8 @@ else
   UBOOT_IMAGE		= $(UBOOT_BIN)
 endif
 
+UBOOT_IMAGE_LS		= $(UBOOT_BUILD_DIR)/$(UBOOT_MACHINE)/u-boot-dtb.bin
+
```

Comment:
For all the changes in this file, again similar comments:

You can't modify how u-boot is built for all the other machines.

I imagine this is a down stream effect of not building the toolchain locally.